doc/bgm.md

```diff
@@ -1,96 +1,109 @@
 # Looping Background Music
 
-With the `Bgm` class you can manage looping of background music tracks with regards to application (or game) lifecycle state changes.
+With the `Bgm` class you can manage looping of background music tracks with regards to application
+(or game) lifecycle state changes.
 
-When the application is paused, terminated, or sent to background, `Bgm` will automatically pause the currently playing music track. Similarly, when the application is resumed, `Bgm` will resume the background music. Manual pause and resume is also supported.
+When the application is paused, terminated, or sent to background, `Bgm` will automatically pause
+the currently playing music track. Similarly, when the application is resumed, `Bgm` will resume the
+background music. Manually pausing and resuming your tracks is also supported.
 
-For this class to function properly, the observer must be registered by calling the following:
+The `Bgm` class is handled by the [flame_audio](https://github.com/flame-engine/flame_audio) library
+so you first have to add that to your dependency list in your `pubspec.yaml` file:
 
-```dart
-Flame.bgm.initialize();
+```yaml
+dependencies:
+  flame_audio: 0.1.0-rc4
 ```
 
-**IMPORTANT Note:** The `initialize` function must be called at a point in time where an instance of the `WidgetsBinding` class already exists. It's safe to assume that this case is true at any point after `runApp` has been called at least once.
-
-In cases where you're done with background music but still want to keep the application/game running, use the `dispose` function to remove the observer.
+For this class to function properly, the observer must be registered by calling the following:
 
 ```dart
-Flame.bgm.dispose();
+FlameAudio.bgm.initialize();
 ```
 
-To play a looping background music track, run:
+**IMPORTANT Note:** The `initialize` function must be called at a point in time where an instance of
+the `WidgetsBinding` class already exists. It's safe to assume that this case is true at any point
+after `runApp` has been called at least once.
```

Comment:
is onLoad a good place as well? maybe we can mention that (or if it's not explain why)